src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRentPrice = 40;
+  const baseAmount = days * dayRentPrice;
+
+  if (days < 3) {
+    return baseAmount;
+  }
+
+  if (days < 7) {
+    return baseAmount - 20;
+  }
+
+  return baseAmount - 50;
```

Comment:
Let's move numberOfDaysToSmallDiscount, same as for large, and amount of discounts